core/slim_sim.cpp

```diff
@@ -6052,33 +6048,26 @@ void SLiMSim::AddIndividualsToTable(Individual * const *p_individual, size_t p_n
 		
 		tsk_id_t tsk_individual;
 		
-		if (p_flags & SLIM_TSK_INDIVIDUAL_FIRST_GEN)
-		{
-			tsk_individual = TSK_NULL;	// not in the table already
-		}
-		else
-		{
-			if (using_std_vector)
-			{
-				// this case has a slow lookup (linear search), but the vector is fast to build
-				auto ind_pos = std::find(remembered_individuals.begin(), remembered_individuals.end(), ped_id);
-				
-				if (ind_pos == remembered_individuals.end())
-					tsk_individual = TSK_NULL;	// not in the table already
-				else
-					tsk_individual = (tsk_id_t)std::distance(remembered_individuals.begin(), ind_pos);
-			}
-			else
-			{
-				// this case has a fash search (hash table), but the unordered_map is slow to build
-				auto ind_pos = remembered_individuals_lookup.find(ped_id);
-				
-				if (ind_pos == remembered_individuals_lookup.end())
-					tsk_individual = TSK_NULL;	// not in the table already
-				else
-					tsk_individual = ind_pos->second;
-			}
-		}
+        if (using_std_vector)
+        {
+            // this case has a slow lookup (linear search), but the vector is fast to build
+            auto ind_pos = std::find(remembered_individuals.begin(), remembered_individuals.end(), ped_id);
+            
+            if (ind_pos == remembered_individuals.end())
+                tsk_individual = TSK_NULL;	// not in the table already
+            else
+                tsk_individual = (tsk_id_t)std::distance(remembered_individuals.begin(), ind_pos);
+        }
+        else
+        {
+            // this case has a fash search (hash table), but the unordered_map is slow to build
```

Comment:
might as well fix the typo while you're modifying this code, fash->fast

Reply:
oh I thot u meant "fashionable search" ;)